services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/_namepsace.py

```diff
@@ -0,0 +1,10 @@
+from models_library.projects_nodes import NodeID
+
+from ...models.schemas.constants import DYNAMIC_SIDECAR_SERVICE_PREFIX
+
+
+def get_compose_namespace(node_uuid: NodeID) -> str:
+    # To avoid collisions for started docker resources a unique identifier is computed:
+    # - avoids container level collisions on same node
+    # - avoids volume level collisions on same node
```

Comment:
what about multiple deployments on same swarm? if a study would be copied from staging to prod, then the node_uuid would be the same in both

Reply:
Well if done via the import feature, it changes the project_id and node_ids in the study. This should not be an issue.

I could append the `SWARM_STACK_NAME`, It should work like that as well.

Reply:
This apparently breaks more that one thing. I will create an issue for it.